lib/js_rails_routes/language/typescript.rb

```diff
@@ -7,14 +7,20 @@ module JSRailsRoutes
   module Language
     class TypeScript < JavaScript
       PROCESS_FUNC = <<~TYPESCRIPT
-        type Value = string | number
+        type Value = string | number | Value[]
```

Comment:
How about this instead, since we don't expect array of array?

```suggestion
        type Value = string | number | (string | number)[]
```

Reply:
Thank you for your feedback. I have made the suggested changes.